inception-project/inception
# #3868 - Allow a recommender to force-create annotations

**What's in the PR**
- Extended recommender API to allow force-accepting suggestions into a document when the document is opened for the first time
- Documented the extension

**How to test manually**
* Create a non-trainable recommender and hook it up to a project - ensure that the recommender sets `<feature-name>_auto_accept` to `on-first-access` on at least one recommendation
* Enable the option "wait for non-trainable recommenders when opening a project" in configuration panel of the recommender list in the project settings
* Open a document and see that the auto-accept recommendation is auto-accepted

**Automatic testing**

**Documentation**
